### PR TITLE
De-fork, bump version number and dependencies

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
De-fork and update de-forked dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,19 +26,24 @@
...
+    "lib"
+  ],
   "dependencies": {
-    "@kartotherian/blend": "^2.0.1",
+    "@mapbox/blend": "^2.0.1",
     "@mapbox/geojson-normalize": "~0.0.1",
-    "@wikimedia/makizushi": "^4.0.0",
-    "agentkeepalive": "^4.0.2",
-    "minimist": "~1.2.0",
+    "@wikimedia/makizushi": "4.1.0",
```
Comment (adamwight) [inline package.json]: Seems fine to automatically pull in minor updates?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## changelog
 
+## 4.0.0
+- Use foked makizushi from wikimedia namspace with maki 7.2.0. icons
```
Comment (adamwight) [inline CHANGELOG.md]: f*rcked

File: test/test.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ var test = require('tape'),
     path = require('path'),
     glob = require('glob'),
     nock = require('nock'),
-    mapnik = require('@kartotherian/mapnik'),
+    mapnik = require('mapnik'),
```
Comment (adamwight) [inline test/test.js:7]: Are you sure we want to land this on `master`?  I think we become incompatible with current forked mapnik if we do this.

Comment (WMDE-Fisch) [reply test/test.js:7]: But isn't the whole idea to ( also ) get rid of the mapnik fork?

File: CHANGELOG.md (hunk 2 above)
Comment (WMDE-Fisch) [reply CHANGELOG.md]: Done.

File: package.json (hunk 1 above)
Comment (WMDE-Fisch) [reply package.json]: Done.

File: test/test.js (hunk 3 above)
Comment (WMDE-Fisch) [reply test/test.js:7]: Just talked about that in the 1on1. :-)

Comment (adamwight) [reply test/test.js:7]: Exciting to see that this is possible!

**Pull request merged**